src/app/App.tsx

```diff
@@ -1,93 +1,43 @@
 import React from "react";
 import { fetchStatusData, StatusDataResponse } from "@bbc/status-data";
+import Header from "./Header";
+import Banner from "./Banner";
+import Capabilities from "./Capabilities";
+import Platforms from "./Platforms";
+import PastIncidents from "./PastIncidents";
 import StatusDataPanel from "./StatusDataPanel";
-
+import RunBooks from "./RunBooks";
 
 const App: React.FC = () => {
   const initState = {
     status: {
       timestamp: "123",
-      results: []
-    }
+      results: [],
+    },
   };
 
   const [data, setData] = React.useState<StatusDataResponse>(initState);
   const fetchData = async (): Promise<void> => {
     const env = "test";
-    const statusData =  await fetchStatusData(env);
+    const statusData = await fetchStatusData(env);
     setData(statusData);
-  }
+  };
   React.useEffect(() => {
     fetchData();
-    });
+`}, []);`
```

Comment:
didn't mean for the ticks to be part of the code:
```suggestion
}, []);
```